README.md

```diff
@@ -425,6 +425,9 @@ such a scenario.
 | CLI Option  | Description       |
 |-------------|-------------------|
 | `--reporter-cli-silent`         | The CLI reporter is internally disabled and you see no output to terminal. |
+
+| `--reporter-cli-show-timestamp` | This prints the local time for each request made. | 
+| `--reporter-cli-show-timestamp-utc` | This prints the UTC time of each request made. |
```

Comment:
I don't feel any need for this option right now. We can just have a single flag to show the timestamps.

Reply:
Alright, I'll fix this